src/sections/Meshmap/Meshmap-design/meshmap-design-integrations.js

```diff
@@ -117,17 +113,48 @@ const MeshmapIntegrationsSection = () => {
 
   return (
     <IntegrationsSectionWrapper>
-      <div className="hero-image" ref={diagramRef} style={{ alignSelf: "center", width: "100%" }}>
-        <IntegrationsImage  alt="integrations-img" className={imageInView ? "diagram-visible" : "diagram-hidden"} />
+      <div
+        className="hero-image"
+        ref={diagramRef}
+        style={{ alignSelf: "center", width: "100%" }}
+      >
+        <IntegrationsImage
+          alt="integrations-img"
+          className={imageInView ? "diagram-visible" : "diagram-hidden"}
+        />
       </div>
       <div className="hero-text">
-        <h1>{Math.ceil(integrations.allMdx.totalCount / 10) * 10}+ Built-in Integrations</h1>
-        <img className="underline-img" src={UnderlineImg}/>
-        <h4>Support for all of your Cloud Native Infrastructure and Applications.</h4>
+        <div>
+          <h1>
+            {Math.ceil(integrations.allMdx.totalCount / 10) * 10}+ Built-in
+            Integrations
+          </h1>
+        </div>
+        <div>
+          <img className="underline-img" src={UnderlineImg} />
+        </div>
+        <div>
+          <h4>
+            Support for all of your Cloud Native Infrastructure and
+            Applications.
+          </h4>
+        </div>
+        <div>
+          <Button
+            secondary
+            style={{ margin: "1.5rem 0 1.5rem 0" }}
+            onClick={function () {
```

Comment:
Thanks for this PR, @Himanshu1801 👍 

Will you please review how this site's standard `<Button>` component is used in other places on the site and tweak the internals here (by removing onClick and navigate) to be consistent with their use of the component?

Reply:
I think `link` can be used instead of onClick since it might be very breaking

Reply:
sure thing

Reply:
LGTM!